web/src/components/molecules/Content/types.ts

```diff
@@ -0,0 +1,19 @@
+import { FieldType } from "@reearth-cms/components/molecules/Schema/types";
+
+export type ItemField = {
+  schemaFieldID: string;
+  type: FieldType;
+  value: any;
```

Comment:
Can you type this better?

Reply:
Same as above, value can be anything.

Reply:
@KaWaite , Backend typed it as `any`, it can be any type from string to an integer to array...etc
![1](https://user-images.githubusercontent.com/78056580/194227264-03d5eadc-e85c-439f-a3bb-cf36d0f77857.png)